aion/web/elm/src/General/Constants.elm

```diff
@@ -24,3 +24,14 @@ panelPath =
 userPath : String
 userPath =
     "#profile"
+
+
+footerContent : List String
+footerContent =
```

Comment:
Is it really necessary to keep such a long footer content?

Reply:
We could leave first two lines and the last one (which would cut content by half), would that be any better?